neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/transaction/impl/RemoteTransactionCoordinatorBuilder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.neo4j.remote.transaction.impl;
+
+import org.hibernate.ogm.datastore.neo4j.remote.impl.RemoteNeo4jDatastoreProvider;
+import org.hibernate.ogm.transaction.emulated.impl.NoopJdbcResourceTransactionCoordinatorOwner;
+import org.hibernate.ogm.transaction.impl.ForwardingTransactionCoordinatorBuilder;
+import org.hibernate.resource.transaction.TransactionCoordinator;
+import org.hibernate.resource.transaction.TransactionCoordinatorBuilder;
+import org.hibernate.resource.transaction.spi.TransactionCoordinatorOwner;
+
+/**
+ * Builder for a Neo4j remote {@link TransactionCoordinator}.
+ *
+ * @author Davide D'Alto
+ */
+public class RemoteTransactionCoordinatorBuilder extends ForwardingTransactionCoordinatorBuilder {
```

Comment:
I'm wondering why this implementation is forwarding. What's the rationale behind it?